src/yarpserver/src/main.cpp

```diff
@@ -5,9 +5,15 @@
  *
  */
 
+#include <stdio.h>
+#include <yarp/os/Network.h>
 #include <yarp/serversql/yarpserversql.h>
 
-int main(int argc, char *argv[]) {
+int main(int argc, char *argv[])
+{
+    // Yarp server must always run using system clock
+    yarp::os::Network::setEnvironment(yarp::os::ConstString("YARP_CLOCK"), yarp::os::ConstString(""));
```

Comment:
I don't know the details on the "full new implementation" mentioned in the PR message, but we plan to eventually replace this with a call to `void Time::useSystemClock()` that always works regardless of when/where it is called?